root/types.js

```diff
@@ -847,6 +852,19 @@ declare type RecordingT = $ReadOnly<{
   +video: boolean,
 }>;
 
+declare type RecordingWithOptionalArtistCreditT = $ReadOnly<{
+  ...AnnotationRoleT,
+  ...CommentRoleT,
+  ...CoreEntityRoleT<'recording'>,
+  ...RatableRoleT,
+  +artist: string,
+  +artistCredit?: ArtistCreditT,
+  +isrcs: $ReadOnlyArray<IsrcT>,
+  +length: number,
+  +related_works: $ReadOnlyArray<number>,
+  +video: boolean,
+}>;
+
```

Comment:
That seems quite redundant with `RecordingT` except for `...ArtistCreditRoleT`, `artist`, and `artistCredit`. But I don’t have no good suggestion here, maybe @mwiencek has one?